### DISCUSSION TITLE
Xiaomi RTCZCGQ11LM update frequence

### DISCUSSION THREAD

**Discussion by johannes-z** [Koenkk/zigbee2mqtt#13498] [Devices]
Hello, I just got an Aqara FP1 (Xiaomi RTCZCGQ11LM).

I tried it with default settings, but the `presence` doesn't update *right away* (after 4 seconds), but only after around 1 minute of no presence. When quickly waving my hands in front of it, `presence` is `true` after 4 seconds, and `false` after > 1minute:
![image](https://user-images.githubusercontent.com/9069888/184104992-3b9d0ed7-b93b-4d41-93f3-e3a003f103dc.png)

Configuration:
![image](https://user-images.githubusercontent.com/9069888/184104841-2883a21e-1634-40e1-a9fa-ff952653201b.png)

Am I missing something here?

**Comment by sorryusernameisalreadytaken** [comment]
Perhaps you should take a look at this workarround if you use homeassistant:
https://community.home-assistant.io/t/new-aqara-human-presence-sensor-fp1/373471/192
